core/amber/src/main/protobuf/edu/uci/ics/amber/engine/architecture/worker/controlcommands.proto

```diff
@@ -48,11 +48,18 @@ message LocalOperatorExceptionV2 {
     string message = 1;
 }
 
+message LinkOrdinal {
+    common.LinkIdentity link_id = 1 [(scalapb.field).no_box = true];
+    int64 port_ordinal = 2;
+}
+
 message InitializeOperatorLogicV2 {
```

Comment:
this control message serves the purpose to initialize an operator logic. similarly, we have "UpdateOperatorLogic" for updating an operator's logic. would it be better to keep port information and layer identity in a separated message if not too much trouble?

Reply:
can refactor in future PR.